src/main/scala/com/typesafe/sbt/bundle/SbtBundle.scala

```diff
@@ -145,8 +154,21 @@ object SbtBundle extends AutoPlugin {
         stageBundle(bundleType.value)
       }.value
     }.value,
+    NativePackagerKeys.dist in Configuration := Def.taskDyn {
+      Def.task {
+        createConfiguration()
+      }.value
+    }.value,
+    NativePackagerKeys.stage in Configuration := Def.taskDyn {
+      Def.task {
+        stageConfiguration
+      }.value
+    }.value,
     NativePackagerKeys.stagingDirectory in Bundle := (target in Bundle).value / "stage",
-    target in Bundle := target.value / "bundle"
+    NativePackagerKeys.stagingDirectory in Configuration := (target in Configuration).value / "stage",
+    target in Bundle := target.value / "bundle",
+    target in Configuration := target.value / "configuration",
+    configurationPath := baseDirectory.value.getAbsolutePath + "/src/main/configuration/"
```

Comment:
This should absolutely be expressed as a path in order to remain platform independent i.e.

```scala
configurationPath := sourceDirectory.value / "configuration"
```

The above also uses the default source directory and is consistent with the way native packager picks up templates etc. from src i.e. there is no distinction between "main" and "test" - there doesn't need to be as this is for deployment...